README.md

```diff
@@ -62,13 +62,36 @@ git clone --recursive https://github.com/clara-genomics/AtacWorks.git
     ```
 ### 3. Tests
 
-Run unit tests:
+Run unit tests to verify that installation was successful
```

Comment:
Say that this is optional.

Reply:
I think it's important to run the unit tests to make sure all the dependencies are installed properly